paasta_tools/utils.py

```diff
@@ -975,6 +975,10 @@ def get_paasta_branch(cluster, instance):
     return 'paasta-%s.%s' % (cluster, instance)
 
 
+def get_paasta_tag(cluster, instance):
+    return get_paasta_branch(cluster, instance)
```

Comment:
I think this is a good start, long term I think we need the type of tag that start/stop does with a timestamp. Otherwise we are moving this same tag over and over which is kinda gross.